pipelines/scripts/check_notebooks_valid.py

```diff
@@ -0,0 +1,60 @@
+from pipelines.scripts.util import Util
+import json
+from typing import Dict, Any
+import logging
+
+
+"""
+    Script to check if notebooks are valid.
+    Run using `python3 pipelines/scripts/check_notebooks_valid.py`
+    You can use `pipelines/scripts/clean_notebooks.py` to clean notebook errors identified by this script
+"""
+
+logging.basicConfig(level=logging.INFO)
+
+
+class InvalidNotebookException(Exception):
+    pass
+
+
+def is_notebook_execution_count_valid(notebook: Dict[str, Any]) -> bool:
+    """
+        Check that the notebook execution count is valid
+
+        :param notebook: The notebook json to analyse
+
+        :return: `True` if the execution count is null, `False` otherwise
+    """
+    return all(
+        [
+            x
+            for x in notebook["properties"]["cells"]
+            if x.get("execution_count", None) is None
```

Comment:
This cell will always return true if there's at least one cell with execution_count == none but it doesn't check all cells 

shouldn't it be like this : 
return all(
    cell.get("execution_count") is None
    for cell in notebook["properties"]["cells"]
)

So you check that every cell has execution_count == None